lib/cc/analyzer/engines_runner.rb

```diff
@@ -5,13 +5,14 @@ module Analyzer
     class EnginesRunner
       NoEnabledEngines = Class.new(StandardError)
 
-      def initialize(registry, formatter, source_dir, config, requested_paths = [], container_label = nil)
+      def initialize(registry, formatter, source_dir, config, requested_paths = [], container_label = nil, partial = false)
```

Comment:
`container_label` is not used. I can remove it but chosen to keep it as the change is not related to this PR.